rfc/system/2704-release-cycle-process-updates.md

```diff
@@ -0,0 +1,107 @@
+# Flyte Versioning & Planning Updates
+
+Hack MD: https://hackmd.io/ZwarNg5iSjeJz2l0YBKIZQ
+
+## Background & Issues
+This is the current [roadmap](https://github.com/flyteorg/flyte/blob/master/rsts/community/roadmap.rst). The core Flyte maintainers team is proposing some updates to it, specifically to the Milestones and Releases section.
+
+We are proposing:
+* Changes to the pace of Flyte releases.
+* Changes to the versioning scheme used across Flyte and its components
+* Updates to the planning process. It has long since been our goal to make Flyte planning a community endeavor.
+
+## Pace
+The regular cadence of monthly releases made sense when Flyte was still an incubating project as APIs were still being upgraded and core experiences were being improved at a rapid pace. As a more stable graduated project, Flyte development needs to ensure stability and backwards compatibility while still delivering substantial feature improvements. We propose moving to quarterly releases, with the understanding that rather than being tied strictly to the calendar, we aim to have substantial features and improvements at each quarter. If features slated for a given release are delayed, then the release will be delayed as well. The increased time will give the Flyte development team more time to beta test each feature and release.
```

Comment:
How do we move towards making releases more frequent?  What are the barriers?  What are the problems?  Resource constraints?  Are these human/labor?  Trying to figure out what is missing from automation to keep or increase release velocity, not slow it down.